### ISSUE TITLE
Barrows plugin NPE

### ISSUE THREAD

**Issue by testing-ongithub** [runelite/runelite#14673]
**Describe the bug**
I spotted a NPE logged in my client.log file while checking to see if anything was logged for an issue I had (I was later informed that that other issue was unrelated to RuneLite).

**To Reproduce**
I have no more information than what is listed in the stacktrace (below). Sorry.

**Expected behavior**
No NPE.

**Environment (please complete the following information):**
 - OS: macOS 12.2
 - RuneLite version: 1.8.10
 - Launcher version: 2.3.0

**Additional context**
Here's the relevant log/stacktrace:
```
2022-02-06 20:46:37 [Client] WARN  n.runelite.client.eventbus.EventBus - Uncaught exception in event subscriber
java.lang.NullPointerException: Cannot invoke "net.runelite.api.ItemContainer.getItems()" because "barrowsRewardContainer" is null
	at net.runelite.client.plugins.barrows.BarrowsPlugin.onWidgetLoaded(BarrowsPlugin.java:181)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:70)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:223)
	at net.runelite.client.callback.Hooks.post(Hooks.java:169)
	at client.dl(client.java:876)
	at au.ae(au.java:4524)
	at by.ka(by.java:12278)
	at client.hf(client.java:7064)
	at client.fo(client.java:3685)
	at client.i(client.java:1843)
	at am.u(am.java:335)
	at am.run(am.java:314)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by Alexsuperfly** [comment]
i believe this is what happens when you open the chest with 0 reward potential and its empty, its innocuous

**Comment by HippolyteMith** [comment]
I try to resolve it in #14675

**Comment by Adam-** [comment]
fixed in #14675
